instrumentation/jsp-2.3/src/test/groovy/JSPInstrumentationForwardTests.groovy

```diff
@@ -525,7 +525,7 @@ class JSPInstrumentationForwardTests extends AgentTestRunner {
           parent()
           operationName expectedOperationName()
           spanKind SERVER
-          errored false
+          errored true
```

Comment:
we should change all `errored` verifications to `status` verifications (issue is fine to track this, it's probably lots of places)

Reply:
Yup - https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/608